docs/examples/general/expressions/expr_type_promotions.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>o perator - &gt; typo</p><p> </p><p>to create pipeline -&gt; to create the pipeline</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>